1_fetch.R

```diff
@@ -118,6 +131,41 @@ p1_targets_list <- list(
   tar_target(
     p1_reaches_sf,
     st_read(p1_reaches_shp)
+  ),
+
+  # fetch prms met data
+  tar_target(
+    p1_prms_met_data_zip_file,
```

Comment:
Above we named targets ending in "_zip" and "_csv" so if we wanted to save ourselves a few keystrokes we could omit the "_file" suffix here and in the prms met data target. This is a very minor comment and I don't have a preference on these target names.